src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1038,6 +1081,12 @@ internal async Task InvokeResponse(InvocationResponse invokeResponse)
             if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out var invocation))
             {
                 var context = invocation.Context;
+
+                if (context.Properties.TryGetValue(ScriptConstants.CancellationTokenRegistration, out CancellationTokenRegistration ctr))
+                {
+                    ctr.Dispose();
```

Comment:
`CancellationTokenRegistration` implements `IAsyncDisposable` as well. Should we call `DisposeAsync` ?

Reply:
Should we also do `context.Properties.Remove(ScriptConstants.CancellationTokenRegistration)` ?